src/background.js

```diff
@@ -31,9 +31,11 @@ chrome.tabs.onAttached.addListener((tabId, data) => {
   // so we can do the same to the next one to open.
   chrome.tabs.get(tabId, tab => {
     if (tab.url.startsWith(URLS.ownOrigin + 'edit.html')) {
-      chrome.windows.get(tab.windowId, {populate: true}, win => {
+      chrome.windows.get(tab.windowId, {
+        populate: true
```

Comment:
Yes please. With the exception of edit.js all other files follow eslint rules (well, a few eslint errors surfaced after its recent update).